lms/services/oauth2_token.py

```diff
@@ -42,6 +43,7 @@ def save(self, access_token, refresh_token, expires_in):
         oauth2_token.expires_in = expires_in
         oauth2_token.received_at = datetime.datetime.utcnow()
 
+    @lru_cache
     def get(self):
```

Comment:
This could be named `get_current` and we'd only need `get` if we ever wanted to retrieve arbitrary tokens.

Out of scope on this PR anywa.